### ISSUE TITLE
Migrate data from 100 theorems

### ISSUE THREAD

**Issue by physikerwelt** [1000-plus/1000-plus.github.io#8]
Currently, the list looks quite empty. I think to bootstrap the repo contents, it would be great to import data from https://www.cs.ru.nl/~freek/100/

**Comment by grunweg** [comment]
Good idea; I agree! Recent pull requests have added data for all Lean formalisations, which includes the 100 theorems.

**Comment by physikerwelt** [comment]
Have these PRs been merged?

**Comment by grunweg** [comment]
The Lean ones: yes!

**Comment by fredrik-bakke** [comment]
I'll note here that there are mismatches between Wikipedia's names for theorems and 100 theorems' names. In particular, this applies to "Bézout's theorem", but there could be more. I mention this, as I see mathlib has linked to the wrong Wikidata identifier for this theorem.

**Comment by grunweg** [comment]
That's a tricky one, great find! Thanks for the mathlib PR; I've left a comment there.
